src/nandini/CodingExams/CodingExam4.java

```diff
@@ -0,0 +1,25 @@
+package nandini.CodingExams;
+//Program 1: WAP to Find Common Element Between Two Arrays
+
+public class CodingExam4 {
+
+	void findDuplicateNumber(int[] input1,int[] input2) {
+		int count =0;
+		for(int i=0; i<input1.length; i++) {
+			for(int j=0; j<input2.length; j++) {
+				if(input1[i] == input2[j]) 
+					count++;
```

Comment:
once we found duplicate number then you should put break, you don't need count variable as well, you directly print the value in if condition